web/src/content/authors.ts

```diff
@@ -0,0 +1,40 @@
+export type Author = {
+  name: string
+  profilePicture: string
+}
+
+export const TomPrestonWerner: Author = {
+  name: 'Tom Preston-Werner',
+  profilePicture:
```

Comment:
I'll follow up with a PR to move all the static assets like images into the repo too.